utils/format/balance.ts

```diff
@@ -88,6 +88,10 @@ export function roundTo(value: number | string, limit = 2) {
   })
 }
 
+export function formatBalanceWithBlank(value: number, decimals?: number, chainSymbol?: string) {
+  return value ? `${parseFloat(format(value, decimals))} ${chainSymbol}` : '--'
+}
```

Comment:
we already have utils for formatting amounts pls use those 

have a look at :
- functions `formatAmountWithRound` or `roundTo` inside `utils/format/balance.ts`
- composable `useAmount`

I'd add a blank param to those